core/src/main/scala/cromwell/core/io/DefaultIoCommand.scala

```diff
@@ -9,22 +9,43 @@ object DefaultIoCommand {
                                   override val destination: Path,
                                   override val overwrite: Boolean) extends IoCopyCommand(
     source, destination, overwrite
-  )
-  case class DefaultIoContentAsStringCommand(override val file: Path, override val options: IoReadOptions) extends IoContentAsStringCommand(file, options)
-  case class DefaultIoSizeCommand(override val file: Path) extends IoSizeCommand(file)
+  ) {
+    customDebug(s"DefaultIoCopyCommand.init source '$source' destination '$destination' overwrite '$overwrite'")
+  }
```

Comment:
TOL are we expecting to see any of these `Default` guys or just the `Gcs` variants?

Reply:
I ran it locally and with my config I see defaults.

Reply:
interesting and not what I expected, I'll have a look at that...

Reply:
![Screen Shot 2020-10-28 at 1 35 23 PM](https://user-images.githubusercontent.com/791985/97474423-78ba7480-1922-11eb-878f-81d876da23d6.png)

Here's one example trace. And my WDL:

```WDL
version 1.0

task t {
    input { File in_file }
    Float in_file_size = size(in_file)
    command <<< echo '~{in_file}' '~{in_file_size}' >>>
    runtime { docker: "ubuntu" }
    output {
        String out = read_string(stdout())
        Float out_size = size(stdout())
    }
}

workflow w {
  call t { input: in_file = "gs://wb-mock-drs-dev/public/some_data.txt" }
  output {
      String out = t.out
      Float out_size = t.out_size
  }
}
```

Reply:
My conf is a mess (I need to clean out a lot of old / temp config), but here it is JIC:

<details>
<summary>application.jes.conf (may be easier to view/copy in github's edit commend mode)</summary>
<pre>
include required("application")

workflow-options.workflow-failure-mode: "ContinueWhilePossible"

backend {
  default = "Papi"
  backendsAllowed = [ "Papi" ]

  providers {
    Local.config.filesystems.gcs.auth = "application-default"
    Papi {
      name = "Papi"
      actor-factory = "cromwell.backend.google.pipelines.v2alpha1.PipelinesApiLifecycleActorFactory"
      config {
        #concurrent-job-limit = 2

        // Google project
        project = "broad-dsde-cromwell-dev"

        // Base bucket for workflow executions
        root = "gs://kshakir-dsde-cromwell-dev/cromwell-executions"

        // Polling for completion backs-off gradually for slower-running jobs.
        // This is the maximum polling interval (in seconds):
        maximum-polling-interval = 600

        genomics {
          // A reference to an auth defined in the ``google`` stanza at the top.  This auth is used to create
          // Pipelines and manipulate auth JSONs.
          auth = "application-default"
          // Endpoint for APIs, no reason to change this unless directed by Google.
          endpoint-url = "https://genomics.googleapis.com/"
        }

        filesystems = {
          gcs.auth = "application-default"
//          drs.auth = "drs-user"
//          http.enabled: true
        }
      }
    }
  }
}

google {
  applicationName = "cromwell-kshakir"
  auths = [
    {
      name = "application-default"
      scheme = "application_default"
    }
    #{
    #  name = "user-service-account"
    #  scheme = "user_service_account"
    #}
    #{
    #  name = "service-account"
    #  scheme = "service_account"
    #  json-file = "/Users/kshakir/src/cromwell/cromwell-service-account.json"
    #}
//    {
//      name = "drs-user"
//      scheme = "user_account"
//      secrets-file = ${HOME}"/.config/gcloud/legacy_credentials/baron.von.testerton@gmail.com/adc.json"
//      secrets-file = ${HOME}"/.config/gcloud/legacy_credentials/kshakir@broadinstitute.org/adc.json"
//    }
  ]
}

call-caching.enabled = true
#call-caching.invalidate-bad-cache-results = false

engine {
  filesystems {
    gcs.auth = "application-default"
//    drs.auth = "drs-user"
    http.enabled: true
  }
}

system {
  new-workflow-poll-rate = 1
}

filesystems.drs.global.config.martha.url = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v3"

system.delete-workflow-files = true

drs.localization.docker-image = "broadinstitute/cromwell-drs-localizer:develop"

# Turning this on while debuging PROD-444
system.file-hash-cache = true
</pre>
</details>